subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/mqtt/Mqtt.kt

```diff
@@ -0,0 +1,26 @@
+package com.egm.stellio.subscription.service.mqtt
+
+object Mqtt {
+
+    object Version {
+        const val KEY = "MQTT-Version"
+        const val V3 = "mqtt3.1.1"
+        const val V5 = "mqtt5.0"
+    }
+
+    object QualityOfService {
+        const val KEY = "MQTT-QoS"
+        const val AT_MOST_ONCE = 0
+//        const val AT_LEAST_ONCE = 1
+//        const val EXACTLY_ONCE = 2
+    }
```

Comment:
why are the other two levels of Qos commented ?

Reply:
Beacause they are not used yet. 
I will remove them.

Reply:
Is this because of MQTT  client limitations ? QoS1 is quite useful in daily use cases

Reply:
No we can support all of them if the property is setup in notifier_info. Just the default property is 0(AT_MOST_ONCE) the others do not need to be defined in the code since the value come from the notifier_info.